components/nats-operator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM connecteverything/nats-operator:0.7.6 as builder
+FROM connecteverything/nats-operator:0.8.0 as builder
```

Comment:
https://github.com/nats-io/nats-operator/issues/323 will block the merging of this PR as of now.

Reply:
The nats-operator image was renamed and is available already in version 0.8.2 but still with the same issues. see my pending PR https://github.com/kyma-project/kyma/pull/11244/